R/assert.R

```diff
@@ -218,6 +226,32 @@ vec_is_vector <- function(x) {
 vec_is_list <- function(x) {
   .Call(vctrs_is_list, x)
 }
+#' @rdname vec_is_list
+#' @export
+vec_is_list_of_vectors <- function(x) {
+  .Call(ffi_is_list_of_vectors, x)
+}
+
+#' @rdname vec_is_list
+#' @export
+vec_check_list <- function(x,
+                           arg = caller_arg(x),
```

Comment:
I have gone ALL IN on requiring empty `...` between required and optional args in functions like these, and personally feel like it is the right way forward in general because it creates safer and more readable user/developer code.